### PR TITLE
Add switch to toolbar (PROJQUAY-4124)

### PR DESCRIPTION
**Pull request by HammerMeetNail**
* Adds switch to toolbar for toggling between old and new ui

image

### CODE REVIEW

**Comment by HammerMeetNail** [comment]
Switch won't redirect to old UI when running through node. Works when compiled to into quay builds.

File: src/index.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import App from './App';
 import reportWebVitals from './reportWebVitals';
 import {RecoilRoot} from 'recoil';
 import '@patternfly/patternfly/patternfly.css';
 import '@patternfly/patternfly/patternfly-addons.css';
 
+// Load App after patternfly so custom CSS that overrides patternfly doesn't require !important
+import App from './App';
```
Comment (Sunandadadi) [inline src/index.tsx:9]: Nice!

File: src/components/header/QuayHeaderToolbar.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+.pf-c-form__label-text {
+  font-weight: 1;
+}
+
+.pf-c-switch {
+  --pf-c-switch__input--checked__label--Color: white;
+  --pf-c-switch__input--not-checked__label--Color: white;
+  --pf-c-switch--Height: 1rem;
+}
```
Comment (Sunandadadi) [inline src/components/header/QuayHeaderToolbar.css:18]: Can you add a line here?

Comment (HammerMeetNail) [reply src/components/header/QuayHeaderToolbar.css:18]: sure thing

**Review by Sunandadadi — APPROVED** [review]
lgtm

**Pull request merged**